demo/mkldnn_quant/README_en.md

```diff
@@ -0,0 +1,183 @@
+
+- For instructions on how to compile the Paddle inference library from source, see [Compile from Source](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/advanced_guide/inference_deployment/inference/build_and_install_lib_cn.html#id12), checkout release/2.0 or develop branch and compile it.
+
+- Users can also download the published [inference Library](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/advanced_guide/inference_deployment/inference/build_and_install_lib_cn.html). Please select `ubuntu14.04_cpu_avx_mkl` latest release or develop version. The downloaded library has to be decompressed and renamed into `fluid_inference` directory and placed in current directory (`/PATH_TO_PaddleSlim/demo/mkldnn_quant/`) for the library to be available. Another option is to set the `PADDLE_ROOT` cmake variable to the `fluid_inference` directory location to link the tests with the Paddle inference library properly.
+
+#### Compile the application
+The source code file of the sample test (`sample_tester.cc`) and the `cmake` files are both located in `demo/mkldnn_quant/`directory.
+
+```
+cd /PATH/TO/PaddleSlim
+cd demo/mkldnn_quant/
+mkdir build
+cd build
+cmake -DPADDLE_ROOT=$PADDLE_ROOT ..
+make -j
+```
+- `-DPADDLE_ROOT` default value is `demo/mkldnn_quant/fluid_inference`. If users download and unzip the library [Inference library from the official website](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/advanced_guide/inference_deployment/inference/build_and_install_lib_cn.html) in current directory `demo/mkldnn_quant/`, users could skip this option.
+
+#### Run the test
+```
+# Bind threads to cores
+export KMP_AFFINITY=granularity=fine,compact,1,0
+export KMP_BLOCKTIME=1
+# Turbo Boost could be set to OFF using the command
+echo 1 | sudo tee /sys/devices/system/cpu/intel_pstate/no_turbo
+# In the file run.sh, set `MODEL_DIR` to `/PATH/TO/FLOAT32/MODEL` or `/PATH/TO/SAVE/INT8/MODEL`
+# In the file run.sh, set `DATA_FILE` to `/PATH/TO/SAVE/BINARY/FILE`
+# For 1 thread performance:
+./run.sh
+# For 20 thread performance:
+./run.sh -1 20
+```
+
+**Available options in the above command and their descriptions are as follows:**
+- **infer_model:** Required. Tested model path. Note that the model parameters files need be saved into multiple files.
+- **infer_data:** Required. The path of the tested data file. Note that it needs to be a binary file converted by `full_ILSVRC2012_val_preprocess`.
+- **batch_size:** Batch size. The default value is 50.
+- **iterations:** Batch iterations. The default is 0, which means predict all batches (image numbers/batch size) in infer_data
+- **num_threads:** Number of CPU threads used. The default value is 1.
+- **with_accuracy_layer:** The model is with accuracy layer or not. Default value false.
+- **use_analysis** Whether to use paddle::NativeConfig to optimize the model. Default value is false.
```

Comment:
`paddle::AnalysisConfig` -> `paddle::NativeConfig`

Reply:
`paddle::NativeConfig` to `paddle::AnalysisConfig` right? Done.